src/main/java/org/opentripplanner/graph_builder/module/osm/OsmDatabase.java

```diff
@@ -803,7 +782,7 @@ private void processMultipolygonRelations() {
             way.addTag(tag, relation.getTag(tag));
           }
         }
-        if (relation.isTag("railway", "platform") && !way.hasTag("railway")) {
+        if (relation.isPlatform() && !way.hasTag("railway")) {
           way.addTag("railway", "platform");
         }
```

Comment:
Does this code work correctly? If I understand this correctly, it now would add this railway=platform tag even if it's a bus platform (i.e. has public_transport=platform tagging).

Reply:
I'm happy to restore the logic to what it was before but I don't really know what this piece of code is useful for. My suspicion is that it is no longer relevant.

If you scroll down a bit you can see similar logic.

Reply:
7aa7476c853f90672e0e72d1a363b91139020f04